src/main/java/de/tum/in/www1/artemis/web/rest/iris/IrisMessageResource.java

```diff
@@ -76,6 +82,10 @@ public ResponseEntity<IrisMessage> createMessage(@PathVariable Long sessionId, @
         irisSessionService.checkHasAccessToIrisSession(session, null);
         var savedMessage = irisMessageService.saveMessage(message, session, IrisMessageSender.USER);
         irisSessionService.requestMessageFromIris(session);
+        savedMessage.setNonce(message.getNonce());
+        CompletableFuture.delayedExecutor(500, TimeUnit.MILLISECONDS).execute(() -> {
```

Comment:
Why do you delay this? This just adds unnecessary latency to the requests.

Reply:
Was introduced in the course of development, and will be removed. However, this only adds latency to other tabs, not the tab that has just sent the message.